pkg/email/email.go

```diff
@@ -369,6 +370,13 @@ func NewClient(config Config, sender sender) (Client, error) {
 	}
 	appTemplates.systemIntakeRetireLCID = systemIntakeRetireLCID
 
+	systemIntakeUnretireLCIDTemplateName := "system_intake_unretire_lcid.gohtml"
+	systemIntakeUnretireLCID := rawTemplates.Lookup(systemIntakeUnretireLCIDTemplateName)
+	if systemIntakeUnretireLCID == nil {
+		return Client{}, templateError(systemIntakeUnretireLCIDTemplateName)
+	}
+	appTemplates.systemIntakeRetireLCID = systemIntakeRetireLCID
```

Comment:
i am guessing it was meant to be:

```suggestion
	appTemplates.systemIntakeUnretireLCID = systemIntakeUnretireLCID
```

Reply:
bfdd1c1e1cd72c79a8df681d96d85eccd104f007